plugins/plugin-github/che-plugin-github-ext-github/src/main/java/org/eclipse/che/ide/ext/github/client/GitHubClientService.java

```diff
@@ -143,9 +162,15 @@ void getPullRequest(@NotNull String owner,
      * @param callback
      *         callback called when operation is done.
      */
-    void createPullRequest(@NotNull String user, @NotNull String repository, @NotNull GitHubPullRequestCreationInput input,
+    void createPullRequest(@NotNull String user,
+                           @NotNull String repository,
+                           @NotNull GitHubPullRequestCreationInput input,
                            @NotNull AsyncRequestCallback<GitHubPullRequest> callback);
```

Comment:
Why this method changed directly instead of deprecation old method and new method addition just like for other methods you did?